decoders/netflow/netflow.go

```diff
@@ -470,8 +476,8 @@ func DecodeMessageIPFIX(payload *bytes.Buffer, templates NetFlowTemplateSystem,
 	/*size = packetIPFIX.Length
 	packetIPFIX.Version = version
 	obsDomainId = packetIPFIX.ObservationDomainId*/
-	flowSet, err := DecodeMessageCommon(payload, templates, packetIPFIX.ObservationDomainId, packetIPFIX.Length, 10)
-	packetIPFIX.FlowSets = append(packetIPFIX.FlowSets, flowSet)
+	flowSets, err := DecodeMessageCommon(payload, templates, packetIPFIX.ObservationDomainId, packetIPFIX.Length, 10)
+	packetIPFIX.FlowSets = flowSets
```

Comment:
Are you sure you want to replace the FlowSets, and not do `packetIPFIX.FlowSets = append(packetIPFIX.FlowSets, flowSet...)`? Scanning through the rest of the code I think this is only ever called on empty `packetIPFIX`s but I'm not 100% sure.